src/test/playermanagertest.cpp

```diff
@@ -129,7 +129,7 @@ class PlayerManagerTest : public MixxxDbTest, SoundSourceProviderRegistration {
 
 TEST_F(PlayerManagerTest, UnEjectTest) {
     // Ejecting an empty deck with no previously-recorded ejected track has no effect.
-    auto deck1 = m_pPlayerManager->getDeck(1);
+    auto deck1 = m_pPlayerManager->getDeck(0);
```

Comment:
our `[ChannelN]` COs are 1-based while this is now 0-based. Is that really an improvement? I'm concerned this will cause confusion and off-by-one errors later.

Reply:
You point out the downside of the change.

The benefit is that it now matches `PlayerManager::groupForDeck(i)` and pevents cases where we have to + 1 before the call and - 1 after it. 

Like here: (I will also place an inline comment) 
```
for (int i = 0; i < m_pPlayerManager->numberOfSamplers(); ++i) {
```

In addition there was a naming clash with Index vs. Number. 

In general we should stick to the rule to use 0 based counting inside the code base and 1 base in all user visible cases. 

Please have a second look. If you still think we should keep the 1 based index here I will fix the issue in that direction.

Reply:
Thank you for the explanation. I see the tension there. I too prefer 0-based indexing. I will take a second look.